asllib/bitvector.mli

```diff
@@ -81,6 +81,10 @@ val to_string : t -> string
 (** Returns a string representing the bitvector, indexed from the right and
     delimited by apostrophes. *)
 
+val to_string_hexa : t -> string
```

Comment:
I feel like some information is lost here regarding the bit vector length. In particular:
- if the length is not a multiple of 4 you can't tell
- in the tests below it feels like there is an extra hex character, e.g. `'0111'` should be `0x7` but is `0x07`

I wonder if a solution is to print the new pseudo-syntax for bit vector literals of specific lengths, e.g. `0x07[:4]` or similar?